server/views/wiki/show.jade

```diff
@@ -0,0 +1,7 @@
+extends ../layout-wide
+block content
+    iframe#wikiFrame(frameborder='no')
+    script.
+        var lang = window.location.toString().match(/\/\w{2}\//);
+        lang = (lang) ? lang[0] : '/en/';
+        $('#wikiFrame').attr('src','http://freecodecamp.github.io/wiki' + lang);
```

Comment:
This works for local dev, but you have to push changes to the "live" ga-pages wiki to test.